dwbb/src/interface/DW_ecc.scala

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_ecc
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    width: Int = 8,
+    chkBits: Int = 5,
+    syndSel: Int = 0
```

Comment:
What's this? should it be Boolean?

Reply:
Yes, it can be Boolean. Have revised this.